### ISSUE TITLE
Saved model issue after export.py

### ISSUE THREAD

**Issue by hakaiderii** [ultralytics/yolov5#4797]
## ❔Question

After training and test successfull using pt model, i  try to export to saved model to run in tensorflow

!python export.py --weights /content/drive/last.pt --img 640 --batch 1 --include saved_model

try to rerun using detect.py for export saved model
!python detect.py --weights /content/last_saved_model/

but i got the issue below. could you please help me

2021-09-15 04:53:25.919877: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:937] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-09-15 04:53:25.920701: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:39] Overriding allow_growth setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
2021-09-15 04:53:25.920755: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1510] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 15030 MB memory:  -> device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:00:04.0, compute capability: 6.0
Importing a function (__inference_tf_bottleneck_10_layer_call_and_return_conditional_losses_17489) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
Importing a function (__inference_tf_bottleneck_4_layer_call_and_return_conditional_losses_44213) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.

**Comment by glenn-jocher** [comment]
@hakaiderii train > export > detect workflow operates correctly, I'm unable to reproduce any errors with this, and you have not shown any errors in your issue above.

<img width="1862" alt="Screenshot 2021-09-15 at 10 53 11" src="https://user-images.githubusercontent.com/26833433/133402939-e8e74d0e-2c04-447c-b0ff-e870850ee5c0.png">

We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

* ✅ **Minimal** – Use as little code as possible that still produces the same problem
* ✅ **Complete** – Provide **all** parts someone else needs to reproduce your problem in the question itself
* ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

In addition to the above requirements, for [Ultralytics](https://ultralytics.com/) to provide assistance your code should be:

* ✅ **Current** – Verify that your code is up-to-date with current GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been resolved by previous commits.
* ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this repository. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) and providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃
